ibis/expr/types/relations.py

```diff
@@ -293,6 +281,19 @@ def _bind_reduction_filter(self, where):
 
         return where.resolve(self)
 
+    def bind(self, *args, **kwargs):
```

Comment:
Needs docstrings and must be covered with tests, but first we should have a consensus on the API behaviour.

Reply:
Tests are there but the docstring is missing.